src/applications/common/schemaform/fields/ArrayField.jsx

```diff
@@ -17,6 +19,20 @@ import { errorSchemaIsValid } from '../validation';
 const Element = Scroll.Element;
 const scroller = Scroll.scroller;
 
+const isEmpty = (collection) => {
```

Comment:
Normally we use the regular function syntax for these functions defined outside of React components. Curious if you had a reason for defining them as arrow functions.

Reply:
Thanks @jbalboni, changed.